x-pack/plugin/blob-cache/src/main/java/org/elasticsearch/blobcache/shared/SharedBlobCacheService.java

```diff
@@ -328,6 +338,7 @@ private CacheEntry(T chunk) {
     private final Runnable evictIncrementer;
 
     private final LongSupplier relativeTimeInNanosSupplier;
+    private final ThrottledTaskRunner evictionsRunner;
```

Comment:
Could we mention `async` somewhere in the name? Otherwise it suggests (at least to me) that all evictions are throttled.

Reply:
Addressed in bd35686ad3dadd6ca452271db00ca6e47241647f